.github/workflows/ci-core.yml

```diff
@@ -109,7 +109,7 @@ jobs:
         if: always()
         uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
         with:
-          name: ${{ matrix.cmd }}_logs
+          name: go_core_tests_${{ matrix.cmd }}_logs
```

Comment:
How does this work?
We already set `matrix.cmd` as:
```yaml
cmd: ["go_core_tests", "go_core_race_tests"]
```

Reply:
Ok my bad I didn't see this But then I don't understand why SQ CI is broken, when the path is properly formatted

Reply:
this was the path that got generated go_core/_race_tests_logs

Reply:
:confused: